modules/heat_conduction/src/materials/AnisoHeatConductionMaterial.C

```diff
@@ -28,12 +28,15 @@ AnisoHeatConductionMaterialTempl<is_ad>::validParams()
   params.addParam<Real>(
       "reference_temperature", 293.0, "Reference temperature for thermal conductivity in Kelvin.");
   params.addParam<std::string>("base_name", "Material property base name.");
-  params.addRequiredParam<std::vector<Real>>("thermal_conductivity",
-                                             "The thermal conductivity tensor values");
+  params.addParam<std::vector<Real>>("thermal_conductivity",
```

Comment:
Numbers are a special case of functions. Those should not be different parameters. In fact the class should couple a material property instead.